source/api/commands/as.md

```diff
@@ -46,22 +45,44 @@ The name of the alias to be referenced later within a {% url `cy.get()` get %} o
 
 ## DOM Element
 
+Aliasing a DOM element and then {% url getting get %} that aliased element.
+
 ```javascript
-cy.route('PUT', /^\/users\/\d+/, 'fixture:user').as('userPut')
-cy.get('form').submit()
-cy.wait('@userPut')
-  .its('url').should('contain', 'users')
+it('disables on click', function () {
+  cy.get('button[type=submit]').as('submitBtn')
+  cy.get('@submitBtn').click().should('be.disabled')
+})
 ```
 
 ## Route
 
+Aliasing a {% url route %} and then {% url waiting wait %} on that aliased route.
+
 ```javascript
 cy.route('PUT', 'users', 'fx:user').as('userPut')
 cy.get('form').submit()
 cy.wait('@userPut')
   .its('url').should('contain', 'users')
 ```
 
+## Fixture
+
+Aliasing a {% url fixture %} and then accessing it via `this`.
+
+```javascript
+beforeEach(function () {
+  cy.fixture('users-admins.json').as('admins');
+});
+
+it('the users fixture is bound to this.admins', function () {
+  cy.log(`There are ${this.admins.length} administrators.`);
+});
+```
```

Comment:
There wasn't an example of aliasing a fixture so I added one.

Reply:
Interesting, yeah I alias fixtures a lot - should def have an example.